### DISCUSSION TITLE
Verwendung auch für Kaifa MA309?

### DISCUSSION THREAD

**Discussion by arkoode** [greenMikeEU/SmartMeterEVNSagemcom-T210-D#34] [General]
Hallo, kann man das Script auch für den Kaifa MA309 verwenden? Vielen Dank!

**Comment by greenMikeEU** [comment]
Hallo,
leider nicht aber ich habe auch eines für den Kaifa
https://github.com/greenMikeEU/SmartMeterEVNKaifaMA309

**Reply by arkoode** [reply]
Hmm schade :-/ schon das habe ich eh gelesen, sehr leicht zu lesen und verständlich, danke dafür :-) 

Hab mir nur gedacht wegen der InfluxDB ists hier halt doch einfach aber gut :-)
